SSA/Projects/InstCombine/lean/g2004h02h23hShiftShiftOverflow_proof.lean

```diff
@@ -0,0 +1,18 @@
+
+import SSA.Projects.InstCombine.LLVM.PrettyEDSL
+import SSA.Projects.InstCombine.TacticAuto
+import SSA.Projects.InstCombine.LLVM.Semantics
```

Comment:
```suggestion
```

These includes are not needed in this file as we purely work on BitVec. Drop these, as Henrik will also not have these includes.